Cargo.toml

```
@@ -77,6 +77,7 @@ chroma-sqlite = { path = "rust/sqlite" }
 mdac = { path = "rust/mdac" }
 wal3 = { path = "rust/wal3" }
 worker = { path = "rust/worker" }
+cli = { path = "rust/cli" }
```

maybe 
```suggestion
chroma-cli = { path = "rust/cli" }
```
for consistency?